src/bundle/Resources/views/admin/object_state/list.html.twig

```diff
@@ -56,16 +64,21 @@
                 {% for object_state in object_states %}
                     <tr>
                         <td class="ez-checkbox-cell">
-                            {{ form_widget(form_states_delete.objectStates[object_state.id], {"disabled": not can_administrate}) }}
+                            {% if can_administrate %}
+                                {{ form_widget(form_states_delete.objectStates[object_state.id]) }}
+                            {% else %}
+                                {% do form_states_delete.objectStates.setRendered %}
```

Comment:
It raises my concerns - it is only marking a field as rendered which means you can prepare a form with any data you want. If `setup:administrate` policy is not validated later by the kernel or the controller handling the for it will allow you to remove any object state without having `setup:administrate` policy.